incubator/hnc/scripts/performance/test.sh

```diff
@@ -33,3 +33,6 @@ do
   kubectl -n hnc-system delete deployment hnc-controller-manager
   . ./scripts/performance/start-controllers.sh
 done
+
+# Clean up topologies
+. ./scripts/performance/clean-up-topologies.sh
```

Comment:
using the `.` here means your're sourcing the file instead of calling it. The subtle difference there means that you could get clashes in variables when sourcing multiple files since your bringing them all into the main context of `test.sh`. If that isn't desired, I don't think we want it sourced. You probably just need to make these scripts executable to not have to source them.

Reply:
Thanks for the explanation! Will make them in to executable.